src/modules/Elsa.Dapper/Dialects/PostgreSqlDialect.cs

```diff
@@ -0,0 +1,21 @@
+using Elsa.Dapper.Abstractions;
+using System.Collections.Generic;
+
+namespace Elsa.Dapper.Dialects;
+
+/// <summary>
+/// Represents a SQL Server dialect.
+/// </summary>
+public class PostgreSqlDialect : SqlDialectBase
+{
+
+    /// <inheritdoc />
+    public virtual string Upsert(string table, string primaryKeyField, string[] fields)
```

Comment:
A method with the same name also exists on the base class. Can we override that one instead?